misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -694,7 +694,11 @@ private static Path getModuleDocPath(Path absolutePkgPath) throws IOException {
     }
 
     private static Path getAbsoluteModulePath(String sourceRoot, Path modulePath) {
-        return Paths.get(sourceRoot).resolve(ProjectDirConstants.SOURCE_DIR_NAME)
+        Path sourcePath = Paths.get(sourceRoot);
+        if(sourcePath.endsWith(modulePath)) {
```

Comment:
Need space after `if`. Exclude `-x check` and see as it might show this error.

Reply:
Fixed.